Source/WebKit/UIProcess/API/C/WKWebsitePolicies.cpp

```diff
@@ -56,27 +56,13 @@ bool WKWebsitePoliciesGetContentBlockersEnabled(WKWebsitePoliciesRef websitePoli
     return toImpl(websitePolicies)->contentBlockersEnabled();
 }
 
-WK_EXPORT WKDictionaryRef WKWebsitePoliciesCopyCustomHeaderFields(WKWebsitePoliciesRef websitePolicies)
+WK_EXPORT WKDictionaryRef WKWebsitePoliciesCopyCustomHeaderFields(WKWebsitePoliciesRef)
 {
-    HashMap<WTF::String, RefPtr<API::Object>> fields;
```

Comment:
Title should clarify that you are not just deprecating but actually de-implementing.